api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -56,3 +56,83 @@ enum FaultInjectionCluster {
   // Faults are injected to ingesss traffic by an upstream service(s).
   FAULTINJECTIONCLUSTER_UPSTREAM = 2;
 }
+
+message Config {
```

Comment:
Maybe call this `HttpFaultConfig` or something that captures that this is HTTP? We assume HTTP deeper in with the http_abort_status, so I don't think this would be protocol agnostic

Reply:
What if we start supporting GRPC status code going forward? One of the approaches would be to add them as part of existing `latency`/`abort` message. Do you think that renaming `Config` to `HttpFaultConfig` still makes sense for this scenario?

Reply:
I don't think it makes a whole lot sense to try to treat gRPC and HTTP as two different things here: gRPC is a RPC mechanism on top of HTTP/2, and so "HTTP faults" like latency trivially apply to gRPC as well. I think if we think of gRPC configuration as a way to specialize the subset of HTTP faults that apply to gRPC requests then we can happily add gRPC fields to a HTTPFault config